pom.xml

```
@@ -9,12 +9,12 @@
     <artifactId>yaml-axis</artifactId>
     <name>Yaml Axis plugin</name>
     <description>Matrix project axis creation and exclusion plugin using yaml file</description>
-    <version>${revision}${changelist}</version>
+    <version>${revision}.${changelist}</version>
```

```suggestion
    <version>${changelist}</version>
```
This is the standard format you want to use.

---

@NotMyFault I chose the "Manually controlled prefix (optional)" method described below because I was concerned about using a large major version.
https://www.jenkins.io/doc/developer/publishing/releasing-cd/#:~:text=Manually%20controlled%20prefix%20(optional)

Is this not recommended as versioning for Jenkins plugins?

If my approach seems odd to Jenkins users, I will choose "Fully automated versioning (recommended)".
https://www.jenkins.io/doc/developer/publishing/releasing-cd/#:~:text=Fully%20automated%20versioning%20(recommended)

---

Major versions will have no impact on this plugin and its consumers, given the Jenkins plugin ecosystem is not bound to semantic versioning. There's no reason to be concerned, you can use fully automated versioning without hesitation.

---

applied!